MiniApp/Classes/core/Display/ViewControllers/Custom Permissions/CustomPermissionsRequestViewController.xib

```diff
@@ -39,6 +39,9 @@
                     <items>
                         <navigationItem id="CQG-Gc-gTL">
                             <barButtonItem key="rightBarButtonItem" title="Allow" id="guU-mA-ZL2">
+                                <userDefinedRuntimeAttributes>
+                                    <userDefinedRuntimeAttribute type="string" keyPath="localizationKey" value="miniapp.sdk.ui.allow"/>
+                                </userDefinedRuntimeAttributes>
```

Comment:
This is gonna save some time for us for sure. Great job finding it.

Reply:
if you take a look back at the sample app storyboards I used the `userDefinedRuntimeAttributes` for rounded corners, but this time I made things more  by using `@IBInspectable` vars

Reply:
Yep. That's really a good thing